doc/rfd/0000/README.md

```diff
@@ -0,0 +1,109 @@
+---
+authors: Jaime Gomes <jaime.gomes@suse.com>
+state: discussion
+---
+
+# Request for Discussion
+
+The goal is to have an constructive discussion within the team about an idea and capture it in a
+document known by RFD.
+
+## Scope
+
+## Metadata
+
+1. authors: owners of the RFD must be listed with their name and email address 
+(John Doe <john.doe@wonder.land>).
+2. state: one of the states discussed below.
+
+## State
+
+1. draft
+2. discussion
+3. published
+4. commited
+5. abandoned
+
+The ```draft``` state indicates that the work is not ready for discussion and that a placeholder
+was set. The RFD owner is puting his thoughs in place before promoting to ```discussion```.
```

Comment:
```suggestion
was set. The RFD owner is putting his thoughts in place before promoting to ```discussion```.
```

Optional gender-neutrality change (since I'm making suggestions on this line anyway):
```suggestion
was set. The RFD owner is putting their thoughts in place before promoting to ```discussion```.
```